data/dbModels/users.js

```diff
@@ -8,13 +8,26 @@ async function create(user) {
   return result[0];
 }
 
-function readBySlackId(slack_mem_id) {
-  return db('users')
-    .where({ slack_mem_id })
-    .first();
+function read(slack_mem_id = null, org_id = null) {
```

Comment:
Are you aware that changes like this (renaming variable names) break things? stick to a name and have it work, if there is going to be a change communicate properly, please